sdk/appcenter/src/main/java/com/microsoft/appcenter/AbstractAppCenterService.java

```diff
@@ -355,56 +354,4 @@ public void run() {
             disabledOrNotStartedRunnable.run();
         }
     }
-
-    /**
-     * Like {{@link #post(Runnable)}} but also post back in U.I. thread.
-     * Use this for example to manage life cycle callbacks to make sure SDK is started and that
-     * every operation runs in order.
-     * <p>
-     * This method will not run the command if the SDK is disabled, the purpose is for internal commands, not APIs.
-     *
-     * @param runnable command to run.
-     */
-    protected synchronized void postOnUiThread(final Runnable runnable) {
```

Comment:
Was that used only in distribute then removed?

Reply:
It was used only in Push.